### PR TITLE
#3806 search mobile overlay fix

### LINKED ISSUE #3806: Weird overlay in mobile while search
If we can fix this too 
Screenshot 2022-08-18 at 12-33-55 KodaDot - Kusama NFT Market Explorer

_Originally posted by @roiLeo in https://github.com/kodadot/nft-gallery/issues/3780#issuecomment-1219326908_

### LINKED ISSUE #3810: Royalty is not visible in white mode
### What happened?

Screenshot 2022-08-23 at 13 58 34

Screenshot 2022-08-23 at 13 58 39

### Please reproduce in steps

- Open NFT with royalty
- turn on white mode
- aaaand it's gone

### Expected Behavior

~

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

_No response_

### Which wallet you are using?

_No response_

### Screenshots

~

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #3824: 📱 Search text not visible in dark mode on mobile
### What happened?

Screen Shot 2022-08-23 at 6 43 13 PM

### Please reproduce in steps

.

### Expected Behavior

.

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

_No response_

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

.

### Relevant log output

```shell
.
```

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3806
- [x] PR closes #3824
- [x] PR closes #3810
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-08-23 at 6 40 41 PM

### CODE REVIEW

File: styles/themes/_light.scss [hunk 1]
```diff
@@ -121,7 +124,7 @@ html.light-mode {
 	.pagination-next,
 	.pagination-link,
 	body .title:not(h2),
-	.auth-avatar .subtitle {
+	.auth-avatar .subtitle, .royalty {
```
Comment (roiLeo) [inline styles/themes/_light.scss:127]: please when adding new rule go to a new line

**Comment by yangwao** [comment]
is code good to go?

**Comment by prachi00** [comment]
> is code good to go?

most probably, fixed the lint issue @roiLeo mentioned

File: components/bsx/Gallery/Item/GalleryItem.vue [hunk 2]
```diff
@@ -81,7 +81,7 @@
                           {{ $t('Unlist') }}
                         </b-button>
                       </div>
-                      <div v-if="nftRoyalties">
```
Comment (Jarsen136) [inline components/bsx/Gallery/Item/GalleryItem.vue:84]: I wonder why you remove this check? What if a nft has no royalties

**Review by Jarsen136 — COMMENTED** [review]
Just one comment.

File: components/bsx/Gallery/Item/GalleryItem.vue (hunk 2 above)
Comment (prachi00) [reply components/bsx/Gallery/Item/GalleryItem.vue:84]: yeah my bad, must have been removed accidentally, added it again, thank you for pointing it out

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Review by roiLeo — COMMENTED** [review]
Something wrong with text & icon color?

Capture d’écran 2022-08-26 à 11 30 11 AM
Capture d’écran 2022-08-26 à 11 29 44 AM

**Comment by yangwao** [comment]
Seems #3824 is not fixed here 👀

**Comment by petersopko** [comment]
> ### What's new?
> * [x]  PR closes
>   [Weird overlay in mobile while search #3806](https://github.com/kodadot/nft-gallery/issues/3806)
>   [📱 Search text not visible in dark mode on mobile  #3824](https://github.com/kodadot/nft-gallery/issues/3824)
>   [Royalty is not visible in white mode #3810](https://github.com/kodadot/nft-gallery/issues/3810)

it order for this to work (link the issues), it needs different formatting, I've updated it for you @prachi00 : 
> PR closes #3806, closes #3824, closes #3810

**Comment by prachi00** [comment]
> 

yeah I did fix it but it got broken again with recent merges, fixed it again now.

**Comment by prachi00** [comment]
> Seems
> 
> * [📱 Search text not visible in dark mode on mobile  #3824](https://github.com/kodadot/nft-gallery/issues/3824) is not fixed here @prachi00  👀


yeah I did fix it but it got broken again with recent merges, fixed it again now.

**Review by DeluneClaire — COMMENTED** [review]
Visibility on both modes looks good, the search text on both modes is working, and the overlay looks normal, not nothing as in the issue. Everything WFM:)

Only have a small question; before you click on the search looks like it's not complete, and it gives a little bit strange feeling, but maybe it's just because of my phone, but it could be there only perhaps "Search... " so the sentence isn't cut. 

If it could be fixed now in this issue, I'll wait for the answer before approving.

search

**Comment by yangwao** [comment]
pay 30

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 46.11 USD/KSM ~ 0.651 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x5bc32ca082b4ebb0f0b4a701ac21c853fa795cdef56c498ac39e5514a1eaa6a7](https://kusama.subscan.io/extrinsic/0x5bc32ca082b4ebb0f0b4a701ac21c853fa795cdef56c498ac39e5514a1eaa6a7)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
> If it could be fixed now in this issue

- https://github.com/kodadot/nft-gallery/issues/3858

**Pull request merged**